1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,19 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let arr = [];
+  let i = 0;
+  while (i % 2 === 0 && n > 0 && n > arr.length) {
```

Comment:
Hello Rahwa
Regarding this function I believe you ought to have another look.
Your output should be one string with commas e.g.  0,2,4

See if you can redo this function with a 'string' in mind. You want to build a string from "" and add even numbers to this string until 'n' is reached.